Kickstarter-iOS/ViewModels/RootViewModel.swift

```diff
@@ -289,3 +289,10 @@ private func first<VC: UIViewController>(_ viewController: VC.Type) -> ([UIViewC
       .flatMap { viewControllers[$0] as? VC }
   }
 }
+
+private func profileController() -> UIViewController? {
+  let showDash = AppEnvironment.current.config?.features["ios_backer_dashboard"] == .some(true)
+  return showDash
+    ? BackerDashboardViewController.instantiate() as UIViewController?
+    : ProfileViewController.instantiate() as UIViewController?
```

Comment:
i think this function can return a `UIViewController` and you can drop the `as UIViewController?`